Sources/StreamChatUI/ChatMessageList/ScrollToLatestMessageButton.swift

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+/// A Button that is used to indicate unread messages in the Message list.
+open class ScrollToLatestMessageButton: _Button, AppearanceProvider {
+    override open func setUpAppearance() {
+        super.setUpAppearance()
+        
+        setImage(appearance.images.scrollDownArrow, for: .normal)
+        backgroundColor = appearance.colorPalette.background8
+        layer.addShadow(color: appearance.colorPalette.shadow)
+        layer.cornerRadius = 20
```

Comment:
This should probably be in `layoutSubivews` and should be something like `bounds.height / 2.0`?